legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-h2/src/test/java/org/finos/legend/engine/persistence/components/BaseTest.java

```diff
@@ -242,39 +242,39 @@ protected void loadStagingDataForBitempWithDeleteInd(String path) throws Excepti
         h2Sink.executeStatement(loadSql);
     }
 
-    protected void loadStagingDataForLoans(String path) throws Exception
+    protected void loadStagingDataForFromOnly(String path) throws Exception
```

Comment:
We can probably replace loans by BitempScheme.
Eg: loadStagingDataForBitempScheme

Reply:
Same comment for the below 3 changes

Reply:
Oops accidentally dropped the word "Bitemporal", will change this to loadStagingDataForBitemporalFromOnly (there is a need to put the "FromOnly" there cos of different staging data for the case "FromAndThrough")